### PR TITLE
Update "decoding" support of Safari

### PR DESCRIPTION
**Pull request by ixkaito**
[This post](https://webkit.org/blog/8216/new-webkit-features-in-safari-11-1/) said "HTML Image Decoding" was a new feature in Safari 11.1.

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: html/elements/img.json [hunk 1]
```diff
@@ -264,7 +264,7 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": false
+                "version_added": "11.1"
               },
               "safari_ios": {
                 "version_added": false
```
Comment (ddbeck) [inline html/elements/img.json]: The post you linked to also reported that this feature is available for Safari for iOS as well:

```suggestion
                "version_added": "11.3"
```

Comment (ixkaito) [reply html/elements/img.json]: Hi @ddbeck ! Thank you for your follow up.

Comment (ddbeck) [reply html/elements/img.json]: Thank you for the PR. Do you plan to apply the change?

Comment (ixkaito) [reply html/elements/img.json]: I'm sorry. I didn't notice the button.

**Pull request merged**